noxfile.py

```diff
@@ -93,12 +93,12 @@ def lint(session):
 @nox.parametrize("pandas_version", ["1.2.0", "1.3.0"])
 def test(session, pandas_version: str):
     session.install("-r", "requirements-dev.txt")
-    session.run("python", "-m", "tests.setup_tests")
     session.install(".")
 
     session.run("python", "-m", "pip", "uninstall", "--yes", "pandas")
```

Comment:
Not sure if we need the `uninstall+install`, what happens if we only keep the `python -m pip install pandas=={pandas_version}`?